apps/central-scan/frontend/src/app_root.tsx

```diff
@@ -476,19 +476,35 @@ export function AppRoot({
     }
   }, [electionJustLoaded, usbDrive.status]);
 
+  // We check configureMutation's status and stop configuration attempts after a failed attempt.
+  // This prevents the app from looping with continued failed configurations.
+  // If the user ejects the USB drive, we reset configureMutation's status so configuration
+  // may be attempted when a new or modified USB drive is inserted.
+  useEffect(() => {
+    if (
+      (configureMutation.isError || configureMutation?.data?.err()) &&
+      usbDrive.status === 'ejected'
+    ) {
+      configureMutation.reset();
```

Comment:
This getting called twice should be okay, right? In React 18 `useEffect` may be called multiple times in development.

Reply:
I believe so, since we check for `ejected` status this shouldn't interrupt a configuration in progress